backend/GetDataAirtable.py

```diff
@@ -0,0 +1,16 @@
+
+from utils.config import Settings
+from utils.helper import GetDataFromAirtable
+
+
+
+
+
+
+def get_settings() -> Settings:
```

Comment:
Instead of creating a separate file, you can add a test endpoint like this to the main.py file:

```
@app.get("/raw-data")
async def raw_data():
data = GetDataFromAirtable(settings.airtable_api_key)
return data

Reply:
I changed have changed it in main.py file.